tools/bionano/bionano_scaffold_edited.xml

```diff
@@ -420,6 +456,15 @@
                     <has_archive_member path=".*/status.txt"/>
                 </assert_contents>
             </output>
+                        <assert_stdout>
```

Comment:
indentation? Are you not using the galaxy-language-server?

Reply:
Yes, I just copied this block from the one which you included.